.github/workflows/cmake.yml

```diff
@@ -23,63 +29,81 @@ jobs:
             name: Windows x86_64
             arch: x86_64
 
-          - os: ubuntu-latest
+          - os: ubuntu-20.04
             container: ubuntu:18.04
             name: Linux x86_64
-            install: libsystemd-dev
+            install: $common_ubuntu_deps build-essential
             arch: x86_64
 
-          - os: ubuntu-latest
+          - os: ubuntu-20.04
             container: ubuntu:18.04
             name: Linux arm
-            install: crossbuild-essential-armhf libsystemd-dev
+            install: $common_ubuntu_deps crossbuild-essential-armhf
             toolchain: Linux-arm.cmake
             cmake_opts: -DCMAKE_BUILD_TYPE=Release
```

Comment:
I noticed that we had not been building our release binaries with openssl. Should we? @ekoby?

Reply:
it is separate discussion